lib/evss/gi_bill_status/gi_bill_status_response.rb

```diff
@@ -98,7 +98,7 @@ def evss_error_key
       end
 
       def text_response?
-        @content_type.include?('text/html')
+        @content_type.include?('text/html') || !@response.body.respond_to?(:key?)
```

Comment:
Is this just extra hardening?  Have you seen responses that were `text/html` that didn't include the response header?

Reply:
Now that the common client is catching responses like `Authentication Failed: Unique ID:...` lower down the GIBS response will be created from that error, and not include a content_type, but also not be a json string (because VAAFI errors).